app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,27 @@
+class UserMailer < ApplicationMailer
```

Comment:
I believe there is an issue with the User emailing, after creating a new user, I do not see a Sign Up Email notification in `/mail` (perhaps there are errors in the `created_user.html.erb`)

Reply:
mmm, maybe the let me check
```ruby
  users_url(@event_user)
```